code/__HELPERS/icons.dm

```diff
@@ -943,6 +943,22 @@ var/global/list/friendly_animal_types = list()
 		return J
 	return 0
 
+/atom/proc/cut_overlays()
+	overlays.Cut()
+	overlays += priority_overlays
+
+/atom/proc/add_overlay(image, priority = 0)
+	if(image in overlays)
+		return
+	var/list/new_overlays = overlays.Copy()
+	if(priority)
+		priority_overlays += image
+		new_overlays += image

```

Comment:
priority_overlays does not actually do anything, it is only there to keep track of which images are priority overlays.